torch/_inductor/select_algorithm.py

```diff
@@ -887,7 +887,7 @@ def tensor_repr(x):
             lines += ["]", f"out = {tensor_repr(out)}", ""]
             return "\n".join(lines)
 
-        benchmark.debug_str = debug_str
+        setattr(benchmark, "debug_str", debug_str) # noqa: B010
```

Comment:
Why add this bug prone behavior? Just add a type ignore or a typing.cast if you really cannot fix it.

Reply:
Ok, just added a type ignore